### PR TITLE
Design/projects page

### PR DESCRIPTION
**Pull request by lankaukk**
**Problem:**
Previously, our /projects page didn't have the ability to sort. 

**Fix:**
Now users can sort their projects by date last edited and title, ascending or descending, by clicking these new buttons. 
I made the `<SortButton>` styled component that cycles through different colors according to its `sortOrder` prop, to indicate  what type of sorting is applied and in what direction, if at all. These buttons have onClick event handlers that point to a `handleSort()` function, which updates the state of `filteredProjects` with a new array of the projects sorted accordingly.

### CODE REVIEW

**Review by balazsbajorics — APPROVED** [review]
nice code!

I tried it and I find it weird that you can set the sorting to "nothing" (which is basically Date Edited), I'd star by Date Edited being active, and I wouldn't allow the control to "deactivate" itself, that should only happen if I click on another control (in our case, the Title)

**Comment by balazsbajorics** [comment]
I noticed that your branch is pretty old! It didn't cause problems for this PR, but in general I'd recommend to periodically pull in the latest changes from master if you are working on a long-lived branch. 
Here's how I do it in the terminal:
I make sure that everything is committed or stashed, then:
`git pull`
`git merge origin/master` 

Or if you use the Github Desktop app,
image
you can select Update from master in the menu bar.

File: website-next/components/projects/cards.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const cardLayout = {
 
 export const cardLayoutStyle = {
   flexBasis: cardLayout.width,
-  flexGrow: 1,
+  // flexGrow: 1,
```
Comment (seanparsons) [inline website-next/components/projects/cards.tsx:16]: Might as well delete this line.

File: website-next/components/projects/projects-page-component.tsx [hunk 2]
```diff
@@ -316,6 +362,43 @@ export class ProjectsPage extends React.Component<EmptyProps, ProjectsState> {
     const visibleProjectCount =
       this.state.filteredProjects.length + this.state.filteredLocalProjects.length
 
+    const reversedProjects = [...this.state.filteredProjects].reverse()
```
Comment (seanparsons) [inline website-next/components/projects/projects-page-component.tsx:365]: We have some utility functions in `array-utils.ts` for creating reversed arrays and for sorting by a comparator, which handle duplicating the array for you.

**Review by seanparsons — APPROVED** [review]

File: website-next/components/projects/projects-page-component.tsx [hunk 3]
```diff
@@ -35,6 +35,45 @@ const FlexNavItem = styled('div')<navItemProps>(
     color: props.selected ? 'white' : colors.default,
   }),
 )
+
+interface sortButtonProps {
```
Comment (Rheeseyb) [inline website-next/components/projects/projects-page-component.tsx:39]:
```suggestion
interface SortButtonProps {
```

Please always use a capital letter at the start of any types and interfaces (aka [PascalCase](https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841))

You'll also need to update the use below

**Review by Rheeseyb — APPROVED** [review]

**Pull request merged**